### PR TITLE
fix: Footer  added

### LINKED ISSUE #6829: Footer missing on Learn layout
https://nodejs.org/en/learn/getting-started/introduction-to-nodejs

### PR DESCRIPTION
**Pull request by Wellitsabhi**
## Description

Added  Footer in 'Learn' and 'About' section

## Validation

Screenshot_20240611_115955
<br>
Screenshot_20240611_115708

## Related Issues

Fixes #6829 

### Check List

- [x] I have read the [Contributing Guidelines](https://github.com/nodejs/nodejs.org/blob/main/CONTRIBUTING.md) and made commit messages that follow the guideline.
- [x] I have run `npm run format` to ensure the code follows the style guide.
- [x] I have run `npm run test` to check if all tests are passing.
- [x] I have run `npx turbo build` to check if the website builds without errors.
- [x] I've covered new added functionality with unit tests if necessary.

### CODE REVIEW

**Comment by AugustinMauroy** [comment]
Ok good, but you need to do something ´cause actually the footer take to much place on mobile screen.

**Comment by Wellitsabhi** [comment]
Ok , will add css for small screens only

**Review by bmuenzenmeyer — DISMISSED** [review]
the footer should not be sticky - it should be the natural last item in the scrolled content

**Comment by Wellitsabhi** [comment]
My finding is  issue only persists where `ArticleLayout` comp is used.
For check - I inserted dummy data in /about , and was working fine , no sticky footer

https://github.com/nodejs/nodejs.org/assets/99867024/f64f3e36-0428-4be2-8412-1be64446afba

**Comment by Wellitsabhi** [comment]
applied external css on \<Withfooter/> , 
 on small screen, it will appear above breadcrumb.  otherwise working noramlly. 
 UI looks good too.

https://github.com/nodejs/nodejs.org/assets/99867024/b0d3808b-16ed-422b-864a-9b9a1d0e2d9b

File: components/Containers/Footer/index.module.css [hunk 1]
```diff
@@ -32,13 +35,33 @@
...
+
+@media (max-width: 640px) {
+  .mobileFooter {
+    display: block;
+  }
+
+  .desktopFooter {
+    display: none;
+  }
+}
```
Comment (abizek) [inline components/Containers/Footer/index.module.css]: I don't mean to nitpick, but could you please follow the existing style guide?

File: layouts/About.tsx [hunk 2]
```diff
@@ -17,10 +20,16 @@ const AboutLayout: FC<PropsWithChildren> = ({ children }) => (
         <main>{children}</main>
 
         <WithMetaBar />
+        <div className={styles.mobileFooter}>
```
Comment (abizek) [inline layouts/About.tsx:23]: The article layout css adds border-top to metabar with nth-child(2) > last-child rule. It has to be modified to nth-child(2) > nth-child(2) to keep the existing behavior. With this PR's change, metabar does not have a border and the footer has a thick border.

Comment (Wellitsabhi) [reply layouts/About.tsx:23]: I have resolved the issue and it's working fine, will be  commiting that code.
Please look into it for further issue.

**Comment by Wellitsabhi** [comment]
@bmuenzenmeyer @abizek @canerakdas 
Can you please review my latest commit  to consider it for merge.
Let me know if it needs any changes.
Thanks.

**Comment by abizek** [comment]
LGTM

**Comment by ovflowd** [comment]
@bmuenzenmeyer I dismissed your review, but feel free to do a re-review :pray:

**Comment by ovflowd** [comment]
Hmmm, the footer is still sticky, and we don't really want that as it means the overall content available for the page itself is diminished...

**Comment by Wellitsabhi** [comment]
@ovflowd  which layout will be good? (below)
1. height- 100vh   ( has better responsivity in medium devices too)

https://github.com/nodejs/nodejs.org/assets/99867024/9cae63a1-9d3e-4538-b1fe-d308371e8bbe

2. height -100%
 
https://github.com/nodejs/nodejs.org/assets/99867024/a30a6fa3-1e29-413b-9a4d-0fe58b35c764


I hope it address sticky issue.

**Comment by bmuenzenmeyer** [comment]
This PR will need to be rebased or recreated now that https://github.com/nodejs/nodejs.org/pull/6850 merged.

**Comment by Wellitsabhi** [comment]
I shall create new PR.
But it would be great if someone can suggest which layout should i follow

**Pull request closed without merging**